src/rules/function-name-case/README.md

```diff
@@ -97,3 +97,125 @@ a {
   background: -WEBKIT-RADIAL-GRADIENT(red, green, blue);
 }
 ```
+
+## Optional options
+
+### `ignoreFunctions: ["/regex/", "non-regex"]`
+
+Ignore case of function names.
+
+For example, with `"lower"`, given:
+
+```js
+["SOME-FUNCTION", "/^get.*$/"]
+```
+
+The following patterns are considered warnings:
+
+```css
+a {
+  color: sOmE-FuNcTiOn();
+}
+```
+
+```css
+a {
+  color: OTHER-SOME-FUNCTION();
+}
+```
+
+```css
+a {
+  color: GetColor();
+}
+```
+
+```css
+a {
+  color: GET_COLOR();
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+a {
+  display: some-function();
```

Comment:
>  I'd also prefer if there was a note in the documentation of these rules that strings are not case sensitive but regexps are. 

Now, I'm confused too. I thought strings *were* case sensitive.

I thought `function-name` is accept because the primary option is `"lower"`. Then the case sensitive string of `FUNCTION-NAME` is explicitly allowed by using the `ignoreFunctions: []` secondary option. Which is why `FuNcTiOn-NaMe` is disallow, as it is neither lowercase nor explicitly added to the secondary options.